service/searcher/service.go

```diff
@@ -61,6 +64,12 @@ func New(config Config) (*Service, error) {
 func (c *Service) Search(request Request) (*Response, error) {
 	c.logger.Log("level", "debug", "message", fmt.Sprintf("searching secret for organization %s, ID %s", request.Organization, request.ID))
 
+	// We never expose the credential-default secret. From the outside, this does not exist.
+	if request.ID == defaultCredentialNameIDPart {
+		c.logger.Log("level", "warn", "message", "attempt to get default credential. Denied.")
```

Comment:
Should we have a log here? Why?
I think it should be a metric if anything, but I don't see it being useful anyway.

Reply:
No strong opinion. I don't think this is a particularly important event.

Reply:
You're right. Removed it.